repose-aggregator/core/repose-core-api/src/main/java/org/openrepose/core/services/RequestProxyService.java

```diff
@@ -31,17 +31,38 @@
     //TODO: this is the most terrible return value ever
     int proxyRequest(String targetHost, HttpServletRequest request, HttpServletResponse response) throws IOException;
 
+    int proxyRequest(String targetHost, HttpServletRequest request, HttpServletResponse response, String connPoolId)
+            throws IOException;
+
     void setRewriteHostHeader(boolean value);
 
+    @Deprecated
```

Comment:
Given that, if there are absolutely no usages of the other methods just remove them, and we'll do the second digit again. Somebody remind to updat e all references to 8.1.1.0 in the documentation